apps/admin-gui/src/app/users/pages/user-detail-page/user-settings/user-settings-associated-users/user-settings-associated-users.component.ts

```diff
@@ -37,38 +37,34 @@ export class UserSettingsAssociatedUsersComponent implements OnInit {
 
   ngOnInit(): void {
     this.loading = true;
-
-    this.route.parent.params
-      .subscribe(params => {
-        this.userId = params["userId"];
-        this.userManager.getUsersBySpecificUser(this.userId).subscribe(associatedUsers => {
-          this.associatedUsers = associatedUsers;
-          this.setAuth();
-          this.loading = false;
-        });
-      });
+    this.user = this.entityStorageService.getEntity();
```

Comment:
on myProfile URL we gets error 
![image](https://user-images.githubusercontent.com/32871167/143566329-af1da121-9a48-4ed0-801b-0911a1eab6b2.png)